packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/CoinControl/UtxoSearch.tsx

```diff
@@ -0,0 +1,81 @@
+import { useCallback, useRef, Dispatch, SetStateAction, ChangeEvent, KeyboardEvent } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { Input, Icon, KEYBOARD_CODE } from '@trezor/components';
+import { useTranslation } from 'src/hooks/suite/useTranslation';
+import { isFeatureFlagEnabled } from '@suite-common/suite-utils';
+import { borders, spacingsPx } from '@trezor/theme';
+
+const Container = styled.div`
+    display: flex;
+    align-items: center;
+    width: 100%;
+`;
+
+const StyledInput = styled(Input)`
+    width: 100%;
+    margin-right: ${spacingsPx.xs};
+    border-radius: ${borders.radii.full};
+
+    input {
```

Comment:
We are trying to not extend styled components anymore (despite we have it in the code pretty much everywhere). So if its not needed we shall figure out a different way. 

@jvaclavik will you please check if this can be styled without extending the styled component? 

Or maybe @jvaclavik if you just can say, maybe in this case its ok.

Reply:
Hey @peter-sanderson, turns out it wasn't really need to extend sytled component. The actual Input component already have a "small" property that behaves just like it.